cms/migrations/0031_alter_homeproductlink_course_product_page.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.2.20 on 2023-08-18 15:12
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("cms", "0030_move_instructor_short_bios_to_long"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="homeproductlink",
+            name="course_product_page",
+            field=models.ForeignKey(
+                blank=True,
+                null=True,
+                on_delete=django.db.models.deletion.SET_NULL,
+                related_name="+",
+                to="wagtailcore.page",
+                verbose_name="Featured Product Page",
```

Comment:
All this actually does is change the verbose name - it was Course-centric prior.